src/cli.js

```diff
@@ -17,7 +17,7 @@ updateNotifier({ pkg: JSON.parse(packageJson) }).notify({ isGlobal: true })
 const cli = meow(
   `
   Usage
-    $ gitmoji
+    $ gitmoji [options] [command]
   Options
```

Comment:
I believe at some point we might want deprecate the options 👀

Reply:
Could happen in a different PR, let's first guarantee everything works as intended.

Reply:
Yeah sure, also we would need to release it as a major